tests/pytorch_tests/model_tests/feature_models/gptq_test.py

```diff
@@ -235,13 +236,92 @@ def compare(self, ptq_model, gptq_model, input_x=None, quantization_info=None):
         self.unit_test.assertTrue(all(w_diff), msg="GPTQ: some weights weren't updated")
 
 
-class SymGumbelLearnRateZeroTest(GPTQBaseTest):
+class UniformGumbelAccuracyTest(GPTQBaseTest):
+
+    def __init__(self, unit_test):
+        super().__init__(unit_test)
+        self.experimental = False
+
+    def get_tpc(self):
+        tpc = super().get_tpc()
+        # Set Uniform quantization
+        tpc.tp_model.default_qco.base_config.weights_quantization_method = QuantizationMethod.UNIFORM
+        return tpc
 
     def get_gptq_config(self):
-        return GradientPTQConfig(5,
-                                 optimizer=torch.optim.Adam([torch.Tensor([])], lr=0),
+        return GradientPTQConfig(100,
```

Comment:
Do we need this number of iterations? Does it increase the test runtime significantly?

Reply:
No. it's pretty quick.

Reply:
reduce to 5